From 01872603ef8ddbc05d6beec104f34b7d894f60e4 Mon Sep 17 00:00:00 2001
From: Scott Bambrough <scott.bambrough@linaro.org>
Date: Wed, 1 Apr 2015 10:01:11 -0400
Subject: [PATCH 05/10] drivers/gpu: arm/utgard: Fix build issue.

The following statement doesn't have consistent behaviour on all machines:
$(wildcard $(src)/linux/license/gpl/*)
On some it actually returns the name of the GPL header file, on others it is a blank string.
Since the driver is in-tree, we can assume GPL compliance.
---
 drivers/gpu/arm/utgard/Kbuild | 19 ++-----------------
 1 file changed, 2 insertions(+), 17 deletions(-)

diff --git a/drivers/gpu/arm/utgard/Kbuild b/drivers/gpu/arm/utgard/Kbuild
index f4b8d59..bf247bc 100644
--- a/drivers/gpu/arm/utgard/Kbuild
+++ b/drivers/gpu/arm/utgard/Kbuild
@@ -23,23 +23,8 @@ MALI_UPPER_HALF_SCHEDULING ?= 1
 MALI_ENABLE_CPU_CYCLES ?= 0
 
 # For customer releases the Linux Device Drivers will be provided as ARM proprietary and GPL releases:
-# The ARM proprietary product will only include the license/proprietary directory
-# The GPL product will only include the license/gpl directory
-ifeq ($(wildcard $(src)/linux/license/gpl/*),)
-    ccflags-y += -I$(src)/linux/license/proprietary
-    ifeq ($(CONFIG_MALI400_PROFILING),y)
-        $(error Profiling is incompatible with non-GPL license)
-    endif
-    ifeq ($(CONFIG_PM_RUNTIME),y)
-        $(error Runtime PM is incompatible with non-GPL license)
-    endif
-    ifeq ($(CONFIG_DMA_SHARED_BUFFER),y)
-        $(error DMA-BUF is incompatible with non-GPL license)
-    endif
-    $(error Linux Device integration is incompatible with non-GPL license)
-else
-    ccflags-y += -I$(src)/linux/license/gpl
-endif
+# The in-tree driver will only use the GPL releases.
+ccflags-y += -I$(src)/linux/license/gpl
 
 ifeq ($(USING_GPU_UTILIZATION), 1)
     ifeq ($(USING_DVFS), 1)
-- 
2.1.4

